core/templates/dev/head/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.ts

```diff
@@ -153,6 +153,21 @@ angular.module('oppia').directive('adminDevModeActivitiesTab', [
           AdminTaskManagerService.finishTask();
         };
 
+        ctrl.loadNewSkillData = function() {
+          AdminTaskManagerService.startTask();
+          ctrl.setStatusMessage('Processing...');
+          $http.post(ADMIN_HANDLER_URL, {
+            action: 'generate_dummy_new_skill_data'
+          }).then(function() {
+            ctrl.setStatusMessage(
+              'Dummy new skill data generated successfully.');
```

Comment:
Dummy skill and questions generated successfully. (In general, always try to be explicit; "data" is quite vague.)

Reply:
Done.